src/schema/rules/sidecars/mrs.yaml

```diff
@@ -0,0 +1,98 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+---
+# MRS Common metadata fields
+MRSScannerHardware:
+  selectors:
+    - modality == "mrs"
+  fields:
+    InstitutionName: recommended
+    InstitutionAddress: recommended
+    InstitutionalDepartmentName: recommended
+    Manufacturer: recommended
+    ManufacturersModelName: recommended
+    DeviceSerialNumber: recommended
+    StationName: recommended
+    SoftwareVersions: recommended
+    MagneticFieldStrength: recommended
+    ReceiveCoilName: recommended
+    ReceiveCoilActiveElements: recommended
+    NumberReceiveCoilActiveElements: optional
+    NumberTransmitCoilActiveElements: optional
+
+MRSSample:
+  selectors:
+    - modality == "mrs"
+  fields:
+    BodyPart:
+      level: optional
+      description_addendum: Corresponds to DICOM Tag 0018, 0015 `Body Part Examined`.
+    BodyPartDetails: optional
+    BodyPartDetailsOntology: optional
+
+MRSSequenceSpecifics:
+  selectors:
+    - modality == "mrs"
+  fields:
+    PulseSequenceType:
+      level: recommended
+      description_addendum: |
+        For example, `"sLASER"`, `"MEGA-PRESS"`, `"EPSI"`, `"Metabolite-cycled MRSI"`.
+    ScanningSequence__mrs: recommended
+    SequenceName: recommended
+    PulseSequenceDetails: recommended
+    MRAcquisitionType: recommended for MRSI
+    WaterSuppression: recommended
+    WaterSuppressionTechnique: optional
+    OuterVolumeSuppression: recommended
+    B0ShimmingTechnique: optional
+    B1ShimmingTechnique: optional
+
+MRSRequiredFields:
+  selectors:
+    - modality == "mrs"
+  fields:
+    ResonantNucleus: required
+    SpectrometerFrequency: required
+    SpectralWidth: required
+    EchoTime: required
+
+MRSRecommendedFields:
+  selectors:
+    - modality == "mrs"
+  fields:
+    NumberOfSpectralPoints: recommended
+    NumberOfTransients:
+      level: optional
+      level_addendum: recommended for SVS and unloc
+    RepetitionTime__mrs: recommended
+    MixingTime: recommended
+    InversionTime: recommended for inversion recovery data
+    FlipAngle: recommended
+    AcquisitionVoxelSize: recommended
+    MatrixSize: recommended for MRSI
+    VolumeAffineMatrix: recommended for MRSI
+    EditTarget: recommended for edited data
+    EditPulse: recommended for edited data
+    EditCondition: recommended for edited data
```

Comment:
Is there a way to identify edited data apart from the presence of these fields?

Reply:
In the NIfTI-MRS format, edited data can be identified by dimension 7 of the data block, which encodes subspectra.

CC'ing @wtclarke here.

Reply:
Not quite. The 5th to 7th dimensions of NIfTI-MRS are flexible in their use. For example, they could be used to store uncombined coils, averages, or editing cases. The use of a dimension is encoded in the `dim_{5,6,7}` keys of the header extension, using a range of pre-defined tags (`DIM_COIL`, `DIM_DYN`, `DIM_EDIT` for the examples of the previous sentence). The order is flexible.

However, this identification process assumes that the data hasn't been processed. For the most common case (MEGA editing), the final output of processing is a single spectrum (constructed from multiple editing cases). This means that the `DIM_EDIT` dimension will have been removed.

Reply:
Okay. These are probably hard to make recommended in the sense that their absence should display a warning. I would suggest just making them optional, as RECOMMENDED would have no force.

If it would be helpful, we could check whether the MRS extension fields are set to DIM_EDIT and warn if the `Edit*` metadata is missing, but if that won't cover all cases of edited data, it's still not generally enforceable.

Reply:
Agreed that these are best set as optional.

Reply:
I've set these fields as `optional`.